projects/ngx-drag-to-select/src/lib/tokens.ts

```diff
@@ -1,5 +1,7 @@
+import { ComponentType } from '@angular/cdk/portal';
```

Comment:
Let's not use this, because the CDK is not a peer dependency of this library and IMO we shouldn't use anything from the CDK for the library.

Reply:
agreed this was silly of me, I just created a version of ComponentType in the models.ts, it is just a Util type anyway that wraps a interface in a constructable, hope this works.